lib/biokbase/catalog/Impl.py

```diff
@@ -1099,21 +1098,19 @@ def log_exec_stats(self, ctx, params):
         """
         # ctx is the context object
         #BEGIN log_exec_stats
-        admin_user_id = ctx['user_id']
-        if not self.cc.is_admin(admin_user_id):
-            raise ValueError('You do not have permission to log execution statistics.')
-        user_id = params['user_id']
-        app_module_name = None if 'app_module_name' not in params else params['app_module_name']
-        app_id = None if 'app_id' not in params else params['app_id']
-        func_module_name = None if 'func_module_name' not in params else params['func_module_name']
-        func_name = params['func_name']
-        git_commit_hash = None if 'git_commit_hash' not in params else params['git_commit_hash']
-        creation_time = params['creation_time']
-        exec_start_time = params['exec_start_time']
-        finish_time = params['finish_time']
-        is_error = params['is_error'] != 0
+        admin_user_id = ctx.get('user_id')
```

Comment:
Any particular reason for the change to the get functions?

Reply:
I figured it was cleaner just to uniformly pass the username and token to the controller but some functions were ok with anonymous calls so I didn't want an index error.